### ISSUE TITLE
Add listing of all examples and what they illustrate

### ISSUE THREAD

**Issue by Remi-Gau** [bids-standard/model-zoo#43]
Maybe in the landing page of the jupyter book

**Comment by adelavega** [comment]
Agree, also FYI @jmumford has plans to create a more guided series of models, and a master pseduo-dataset that can demonstrate a great range of statsmodel concepts. These may become the primary walk through, followed by a more miscellaneous (and open) collection  of examples.
